src/libpriv/rpmostree-script-gperf.gperf

```diff
@@ -1,45 +0,0 @@
-%{
-#include "config.h"
-#include "rpmostree-scripts.h"
-%}
-struct RpmOstreePackageScriptHandler;
-%language=ANSI-C
-%define slot-name package_script
-%define hash-function-name rpmostree_script_gperf_hash
-%define lookup-function-name rpmostree_script_gperf_lookup
-%readonly-tables
-%omit-struct-type
-%struct-type
-%includes
-%%
-glibc.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-# We take over depmod/dracut etc.  It's `kernel` in C7 and kernel-core in F25+
-kernel.posttrans, RPMOSTREE_SCRIPT_ACTION_IGNORE
-kernel-core.posttrans, RPMOSTREE_SCRIPT_ACTION_IGNORE
-# Legacy workaround
-glibc-headers.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-# workaround for old bug?
-coreutils.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-# Looks like legacy...
-ca-certificates.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-libgcc.post, RPMOSTREE_SCRIPT_ACTION_IGNORE
-setup.post, RPMOSTREE_SCRIPT_ACTION_IGNORE
-pinentry.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-fedora-release.posttrans, RPMOSTREE_SCRIPT_ACTION_IGNORE
-# These add the vagrant group which IMO is really
-# a libvirt-user group
-vagrant.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-vagrant-libvirt.prein, RPMOSTREE_SCRIPT_ACTION_IGNORE
-bash.post, RPMOSTREE_SCRIPT_ACTION_TODO_SHELL_POSTTRANS
```

Comment:
This was effectively the only user of that enum variant, which I can guess can be dropped now (and perhaps the whole enum too?).

Reply:
Yeah, well spotted!  There's no strong reason to have a separate enum just for this over a comment.